pkg/plugins/manager/client/client.go

```diff
@@ -165,3 +182,18 @@ func (s *Service) plugin(ctx context.Context, pluginID string) (*plugins.Plugin,
 
 	return p, true
 }
+
+func (s *Service) attachJWT(ctx context.Context, pluginCtx backend.PluginContext) context.Context {
+	if pluginCtx.User == nil {
+		return ctx
+	}
+
+	token, err := s.jwtAuthService.Generate(pluginCtx.User.Login, pluginCtx.PluginID)
+	if err != nil {
+		return ctx
+	}
+
+	md := metadata.New(map[string]string{})
+	md["authorization"] = []string{"Bearer " + token}
```

Comment:
What is the context behind setting value as a string array vs just a string of the JWT? 

Do we expect the user to have multiple JWT in a session or something?